test/pod_annotations_capability.bats

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+  setup_test
+  CNI_PLUGIN_NAME="log_cni_plugin"
+}
+
+function teardown() {
+  cleanup_test
+}
+
+function run_pod() {
+  crictl runp "$TESTDATA"/sandbox_config.json
+}
+
+function delete_pod() {
+  id=$(crictl runp "$TESTDATA"/sandbox_config.json)
+  crictl stopp "$id"
+  crictl rmp "$id"
+}
+
+function prepare_cni_plugin() {
```

Comment:
instead of copying these helpers, can you put them in test/helpers.bash and reference them from each network.bats and here?